lua/nvim_lsp/pyls.lua

```diff
@@ -10,19 +10,55 @@ skeleton.pyls = {
     filetypes = {"python"};
     root_dir = function() return cwd end;
     log_level = lsp.protocol.MessageType.Warning;
+    settings = {};
   };
   -- on_new_config = function(new_config) end;
   -- on_attach = function(client, bufnr) end;
   docs = {
     description = [[
-https://github.com/palantir/python-language-server
+    https://github.com/palantir/python-language-server
```

Comment:
One final nit, I swear:

Remove the indentation on this docs string and use code fence for the settings part, e.g.

    ````
    settings = {
      ...
    }
    ```

Reply:
No problem! Sorry for the back-and-forth. Are the four backticks at the start on purpose? Should I give a `lua` specifier?

Reply:
no it was because I was a dumb, and yeah a lua specifier should be fine.

At least now, you probably know what to do for other servers if you decide to add them :P and I can point people at your pyls.lua and say "just copy this style."